### PR TITLE
Added Blob Project Presences

### PR DESCRIPTION
**Pull request by Lydzx**
image
image
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy error

**Review by Lydzx — COMMENTED** [review]
Done.

File: websites/B/Blob Project/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    "url": "blob-project.com",
+    "description": {
+      "en": "Hello, welcome to Blob Project. We are from Jakarta, Indonesia. We create an amazing project and are trusted by thousands of people."
+    },
+    "service": "Blob Project",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/xUh0qSN.png",
+    "thumbnail": "https://i.imgur.com/2m91rBI.png",
+    "color": "#7289DA",
```
Comment (Bas950) [inline websites/B/Blob Project/dist/metadata.json:15]: thumbnail should be a promotional banner or a screenshot of the website homepage.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Lydzx — COMMENTED** [review]
Done.

File: websites/B/Blob Project/dist/metadata.json (hunk 1 above)
Comment (Lydzx) [reply websites/B/Blob Project/dist/metadata.json]: Done.

**Review by Lydzx — COMMENTED** [review]
All done.

**Review by ririxichan — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**